qutebrowser/browser/qutescheme.py

```diff
@@ -201,6 +201,31 @@ def qute_bookmarks(_url):
     return 'text/html', html
 
 
+def _tab_fields_to_tabs_page_info(fields):
+    return (fields['title'], fields['current_url'])
+
+
+@add_handler('tabs')
+def qute_tabs(_url):
+    """Handler for qute://tabs. Display information about all open tabs."""
+    tabs = {}
+    for win_id in objreg.window_registry:
```

Comment:
It's possible that you get windows which are in mid-shutdown here. To avoid them, you should add something like:

```python3
for window, win_id in objreg.window_registry.items():
    if sip.isdeleted(window):
        continue
```